include/libopencm3/stm32/fdcan.h

```diff
@@ -0,0 +1,793 @@
+/** @defgroup fdcan_defines FDCAN Defines
+
+@ingroup STM32G_defines
+
+@brief <b>libopencm3 Defined Constants and Types for STM32 FD-CAN</b>
+
+@author @htmlonly &copy @endhtmlonly 2021 Eduard Drusa <ventyl8 at netkosice dot sk>
+
+LGPL License Terms @ref lgpl_license
+*/
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2021 Eduard Drusa <ventyl86@netkosice.sk>
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+
+#ifndef LIBOPENCM3_FDCAN_H
+#define LIBOPENCM3_FDCAN_H
+
+#include <libopencm3/stm32/memorymap.h>
+#include <libopencm3/cm3/common.h>
+
+/** @{ */
+
+/* FDCAN block base addresses. Used in functions to identify FDCAN block being manipulated. */
+
+/** @defgroup fdcan_block FDCAN block base addresses
+ * @{
+ */
+#define CAN1							FDCAN1_BASE
+#define CAN2							FDCAN2_BASE
+#define CAN3							FDCAN3_BASE
```

Comment:
do any parts have both bxcan and fdcan? is this a safe idea to just have "can" be the name? (not an expert in this field, just curious on this one, as we're using fdcan for everything else in this file)

Reply:
I am not aware of any parts which would have both BXCAN and FDCAN. It probably doesn't make any sense as FDCAN is generally backwards compatible with CAN 2.0 and is more capable, than BXCAN.